src/Serilog.Sinks.MSSqlServer/LoggerConfigurationMSSqlServerExtensions.cs

```diff
@@ -45,14 +46,14 @@ public static class LoggerConfigurationMSSqlServerExtensions
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
             int batchPostingLimit = MSSqlServerSink.DefaultBatchPostingLimit,
             TimeSpan? period = null,
-            IFormatProvider formatProvider = null)
+            IFormatProvider formatProvider = null, bool utcTimestamp = true)
```

Comment:
By forcing it to true, you do change the default behaviour. Would an opt in not be better?